lib/src/definitions.dart

```diff
@@ -141,6 +141,70 @@ class DirectionsMessage {
   });
 }
 
+class Camera {
+  /// Set the [zoom] to the desired value. In case [zoom] is out of the range of values we calculate internally,
+  /// a fallback minimum or maximum value will be set.
+  ///
+  /// Value defaults to and internally calculated intermediate.
+  double? zoom;
+
+  /// Set the [bearing] to a determined degree.
+  ///
+  /// value defaults to 0°.
+  double? bearing;
```

Comment:
Na PR do map-viewer xa o apuntei, aquí ollo igualmente con como pasamos este bearing.
No caso de Flutter o obxecto Location leva encapsulado un obxecto Bearing.
Quizais debería chamarse Angle igual que no SDK e ser utilizado para ambos, bearing e pitch?

CC/ @fsvilas, @frandieguez, @adricanedo, @dariovillar

Reply:
Como comentei na PR do viewer, e máis escalable tipalo como **Angle** como temos no SDK para que o dia de mañan non teñamos que romper api porque queremos empezar a usar radians ou outra unidade.

Reply:
PD: vou unificar Bearing e Angle xa que son 2 clases que son practicamente o mismo pero con nome distinto